src/Generators/MigrationGenerator.php

```diff
@@ -57,7 +57,7 @@ public function output(array $tree): array
                         continue;
                     }
 
-                    $path = $this->getPivotTablePath($pivotTable, $sequential_timestamp);
+                    $path = $this->getPivotTablePath($pivotTable, $sequential_timestamp->addSecond());
```

Comment:
This should not be removed. It's likely why the test is failing.

When using constraints, I see why the pivot table would need to be created _after_ the other tables. However, there's really no way to guarantee the order. The best option would be to create all pivot tables _last_.

Reply:
How would that be done? The pivot migration gets generated through the `belongsToMany: Model` option and unfortunately the pivot table gets generated before the secondary table which causes this to always fail.. Am I missing something? 🤔

"The best option would be to create all pivot tables last.,, 

I considered this too but I was to unfamiliar with the codebase to make such a major adjustment, therefore my decision.

Reply:
No worries. Just revert this line for now since so the original test passes and I'll clean it up after merge as it is a related, but separate issue.

Reply:
Gave it some thought and I might have come up with a solution. A separate migration could be generated at the end which will solely be to create constraints. For example;
```
        Schema::table('comment_post', function (Blueprint $table) {
            $table->foreign('comment_id')->references('id')->on('comments');
            $table->foreign('post_id')->references('id')->on('posts');
        });
```